src/proof/resolution_bitvector_proof.h

```diff
@@ -0,0 +1,129 @@
+/*********************                                                        */
+/*! \file resolution_bitvector_proof.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Liana Hadarean, Mathias Preiner, Guy Katz
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Bitvector proof
+ **
+ ** Bitvector proof
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__PROOF__RESOLUTIONBITVECTORPROOF_H
+#define __CVC4__PROOF__RESOLUTIONBITVECTORPROOF_H
+
+#include <iostream>
+#include <sstream>
```

Comment:
Instead of including those two headers, we should be able to just include `iosfwd` in the header and then the required headers in the source file. `iostream` is large and slow to compile, so we need to be a bit careful with including it in the header.